src/sql/workbench/browser/designer/designer.ts

```diff
@@ -941,6 +980,103 @@ export class Designer extends Disposable implements IThemable {
 		return component;
 	}
 
+	private addTableDesignerTableButtons(container: HTMLElement, tableProperties: DesignerTableProperties,
+		path: DesignerPropertyPath, view: DesignerUIArea): void {
```

Comment:
As per our discussion, the new features should be based on the table property, canAddRows, CanRemoveRows, similarly, you should add canInsertRows, canMoveRows, but i don't see these new options.